scripts/run-typewriter-clean-metadata.ps1

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+
+Write-Host "Running Typewriter to clean the metadata..."
+Push-Location -Path $env:TypewriterDirectory
+
+Write-Host "Typewriter.exe: $env:TypewriterExecutable `nInput metadata: $env:InputMetadataFile `n output path: $env:OutputPath"
```

Comment:
Can we remove this given that I have the logger enabled in Typewriter.exe to print out the arguments now?